### ISSUE TITLE
Porting to Windows/Linux

### ISSUE THREAD

**Issue by PaulBGD** [jingweno/hacker-menu#18]
What would be required to port this to Windows/Linux?

**Comment by ngenerio** [comment]
I'm on Windows and it didn't build on it due to `nslog` package. When I removed the `nslog` package and it's related code, I built it and it worked.

**Comment by PaulBGD** [comment]
Yeah I'm seeing the same thing, let me try removing nslog.

**Comment by PaulBGD** [comment]
I did get it working, although it spans multiple monitors unless I add a few icons. This works great nonetheless!

**Comment by ngenerio** [comment]
Spans multiple monitors??

**Comment by PaulBGD** [comment]
Yeah, half is on one and half is on the other. Adding more icons to the right of the hacker menu icon temporarily fixed this.

**Comment by ngenerio** [comment]
Alright!

**Comment by jingweno** [comment]
I confirmed that `nslog` is causing issues when compiling on Windows. We'd need to a find an alternative of it. However, the menu doesn't pop up on Windows 10, will need to investigate more:

<img width="479" alt="screen shot 2015-09-17 at 8 45 00 pm" src="https://cloud.githubusercontent.com/assets/169064/9951578/0120d9de-5d7d-11e5-8aae-4727c75f4d8f.png">

**Comment by PaulBGD** [comment]
Showed up fine for me, maybe you need to use the official Windows 10?

**Comment by onbjerg** [comment]
@PaulBGD Shots fired!

**Comment by onbjerg** [comment]
I am currently writing a packaging script for Windows (and Linux) in #25. It builds for me on Windows 10 without issues.

Care to test for me?

**Comment by PaulBGD** [comment]
```> hacker-menu@1.1.3 build c:\Users\Paul\hacker-menu
> babel src -D -d dist --compact true

src\client\main.js -> dist\client\main.js
src\client\menu.js -> dist\client\menu.js
src\client\spinner.js -> dist\client\spinner.js
src\client\story.js -> dist\client\story.js
src\client\story_box.js -> dist\client\story_box.js
src\client\story_list.js -> dist\client\story_list.js
src\index.js -> dist\index.js
src\logger.js -> dist\logger.js
src\model\read_cache.js -> dist\model\read_cache.js
src\model\story_manager_status.js -> dist\model\story_manager_status.js
src\model\story_type.js -> dist\model\story_type.js
src\server\auto_update_manager.js -> dist\server\auto_update_manager.js
src\server\story_manager.js -> dist\server\story_manager.js
src\server\tray_manager.js -> dist\server\tray_manager.js
'.' is not recognized as an internal or external command,
operable program or batch file.```

**Comment by PaulBGD** [comment]
If we're porting to Windows, creating an installer with https://github.com/atom/grunt-electron-installer would be great.

**Comment by jingweno** [comment]
You can now run `script/package --all` to build for all platforms. But somehow the menu didn't pop up for Windows 10. I created an issue: https://github.com/maxogden/menubar/issues/67.

**Comment by onbjerg** [comment]
The menu pops up on Windows 10, but when I click outside the menu in order to close it, it just goes blank. In order to make it work again, I have to right click on a process in my task bar called Electron and click "Close Window".
